09-Time-Series-Data.ipynb

```diff
@@ -0,0 +1,786 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from __future__ import print_function, division"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Handling Data over time\n",
+    "\n",
+    "There's a widespread trend in solar physics at the moment for correlation over actual science, so being able to handle data over time spans is a skill we all need to have. Python has ample support for this so lets have a look at what we can use.\n",
+    "\n",
+    "<section class=\"objectives panel panel-warning\">\n",
+    "<div class=\"panel-heading\">\n",
+    "<h2><span class=\"fa fa-certificate\"></span> Learning Objectives </h2>\n",
+    "</div>\n",
+    "<br/>\n",
+    "<ul>\n",
+    "<li> Understand and use SunPy lightcurve. </li>\n",
+    "<li> Create a pandas dataframe. </li>\n",
+    "<li> Utilise the datetime pakage </li>\n",
+    "<li> Use the pandas dataframe to plot the data within it. </li>\n",
+    "</ul>\n",
+    "<br/>\n",
+    "</section>\n",
+    "\n",
+    "\n",
+    "\n",
+    "## SunPy Lightcurve\n",
+    "\n",
+    "SunPy provides a lightcurve object to handle this type of time series data. The module has a number of instruments associated with it, including:\n",
+    "\n",
+    "* GOES XRS LightCurve\n",
+    "* SDO EVE LightCurve for level 0CS data\n",
+    "* Proba-2 LYRA LightCurve\n",
+    "* NOAA Solar Cycle monthly indices.\n",
+    "* Nobeyama Radioheliograph Correlation LightCurve.\n",
+    "* RHESSI X-ray Summary LightCurve.\n",
+    "\n",
+    "We're going to examine the lightcurves created by a solar flare on June 7th 2011.\n",
+    "\n",
+    "Lets begin with the import statements:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from __future__ import print_function, division\n",
+    "import numpy as np\n",
+    "import sunpy\n",
+    "from sunpy import lightcurve as lc\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline\n",
+    "\n",
+    "import warnings\n",
+    "warnings.filterwarnings('ignore')"
```

Comment:
May be worth keeping these import statements in the student copy for time-saving purposes.